components/database/mariadb-101/patches/01-cfi-issue.patch

```diff
@@ -0,0 +1,22 @@
+--- mariadb-10.1.17/mysys/my_context.c.~1~	2016-07-27 19:23:07.480630772 +0300
```

Comment:
The patch for search path is not needed more. This fix already is on MariaDB official source: https://github.com/MariaDB/server/blob/10.1/mysys/my_default.c#L1230